neuralmonkey/encoders/attentive.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=too-few-public-methods
+class Attentive(object):
+    def __init__(self, attention_type, **kwargs):
+        self._attention_type = attention_type
+        self._attention_kwargs = kwargs
+
+        assert hasattr(self, "name")
+        assert hasattr(self, "_padding")
+        assert hasattr(self, "_attention_tensor")
```

Comment:
Jestli myslíš, že je to tak správnější, tak já to tak s radostí přepíšu, ale sám nevím.